src/fixate/drivers/dmm/fluke_8846a.py

```diff
@@ -100,11 +81,15 @@ def trigger(self):
         self._write("*TRG")  # Send trigger to instrument
         self._is_error()  # Catch errors. This might slow things down
 
-    def measurement(self):
+    def measurement(self, delay=False):
         """
         Sets up DMM triggering, creates list of measurements from the read buffer
+
+        delay: If True, will wait for self.measurement_delay seconds before triggering a measurement.
         returns: a single value as a float
         """
+        if delay:
+            time.sleep(self.measurement_delay)
```

Comment:
how about this. Then you can override the default for a single measurement. And if you want a delay on every measurement, you don't have to pass in `delay = True` everywhere.
```python
def measurement(self, delay=None):
    if delay is None:
        delay = self.measurement_delay
    
    if delay > 0:
        time.sleep(delay)
````

Reply:
Good point!